<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="summaryPage">
     <attribute name="title">
      <string>Yhteenveto</string>
     </attribute>
     <widget class="QTableWidget" name="meatSharedTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>301</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="groupSummaryTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>301</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="meatSharedLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Jaetut lihat ryhmittäin</string>
      </property>
     </widget>
     <widget class="QLabel" name="groupSummaryLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ryhmän tiedot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="summaryRefreshPushButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Päivitä</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="killPage">
     <attribute name="title">
      <string>Kaato</string>
     </attribute>
     <widget class="QComboBox" name="shotByComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="shotByLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Kaataja</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="Finnish" country="Finland"/>
      </property>
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sharePage">
     <attribute name="title">
      <string>Lihanjako</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="licensePage">
     <attribute name="title">
      <string>Luvat</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="maintenancePage">
     <attribute name="title">
      <string>Ylläpito</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
